src/myuw-notifications.js

```diff
@@ -246,6 +251,8 @@ export class MyUWNotifications extends HTMLElement {
     componentReady(notifications) {
       // Check for notifications
       if (typeof notifications === 'object' && notifications.length > 0) {
+        this.$count.innerHTML = notifications.length;
```

Comment:
This works for making the notification count apparent to users visually looking at the bell icon. Nice! However, I don't see how this provides equivalent access users relying upon a screen reader -- at best it'll be an integer in a span without apparent context? I think we need this notifications.length to feed into an aria description or label or something?

Reply:
Instead of adding aria-label to the span element, I dynamically change aria-label of the bell button (added a video on Teams review channel).